docker/dpcpp/configure.sh

```diff
@@ -8,4 +8,5 @@ cmake . -G Ninja -B build \
     -DCMAKE_CXX_COMPILER=/sycl/bin/clang++ \
     -DCMAKE_BUILD_TYPE=Release \
     -DSYCL_CTS_ENABLE_FULL_CONFORMANCE=0 \
+    -DSYCL_ENABLE_EXT_ONEAPI_PROPERTY_LIST_TESTS=0 \
```

Comment:
Is this intentional? Presumably DPC++ supports this extension?

Reply:
These tests are off by default for all compilers. so removed this too.